it/mail/chrome/messenger/messenger.properties

```diff
@@ -439,36 +439,36 @@ megaByteAbbreviation2=%.*f MB
 gigaByteAbbreviation2=%.*f GB
 
 # Error message if message for a message id wasn't found
-errorOpenMessageForMessageIdTitle=Errore all'apertura dell'identificativo del messaggio
-errorOpenMessageForMessageIdMessage=Messaggio per l'identificativo-messaggio %S non trovato
+errorOpenMessageForMessageIdTitle=Errore all'apertura dell'ID messaggio
+errorOpenMessageForMessageIdMessage=Messaggio per l'ID messaggio %S non trovato
 
 # Warnings to alert users about phishing urls
-confirmPhishingTitle=Possibile frode via e-mail
+confirmPhishingTitle=Avviso frode e-mail
 #LOCALIZATION NOTE %1$S is the brand name, %2$S is the host name of the url being visited
-confirmPhishingUrl=%1$S crede che questo messaggio sia una frode. Probabilmente i link all'interno del messaggio stanno cercando di far aprire delle pagine diverse da quelle che sembrerebbe. Si desidera comunque visitare %2$S?
-confirmMalwareUrlTitle=Reported Attack Site!
+confirmPhishingUrl=%1$S crede che questo messaggio sia una frode. Probabilmente i link all'interno del messaggio stanno cercando di far aprire delle pagine diverse. Sei sicuro di volere visitare %2$S?
+confirmMalwareUrlTitle=Sito di attacco segnalato!
 #LOCALIZATION NOTE %1$S is the host name of the url being visited
-confirmMalwareUrl=Questo sito Web è stato segnalato come pericoloso. Desideri davvero visitare %1$S?
+confirmMalwareUrl=Questo sito web è stato segnalato come pericoloso. Sei sicuro di volere visitare %1$S?
 
 # Check for Updates
 updatesItem_default=Controlla gli aggiornamenti…
 updatesItem_defaultFallback=Controlla gli aggiornamenti…
 updatesItem_default.accesskey=C
-updatesItem_downloading=Ricezione %S…
-updatesItem_downloadingFallback=Ricezione aggiornamento…
+updatesItem_downloading=Download %S in corso…
```

Comment:
CORRECTION NEEDED.

I would add the word "di" as follows: "Download di %S in corso...".